src/liquid/app/header.html

```diff
@@ -0,0 +1,42 @@
+<nav class="navbar">
+    <div class="navbar-branding" id="logo">
+        <a href="/"><img src="{{ logo }}" /></a>
+    </div>
+    <div class="navbar-branding" id="title"><a>User Management</a></div>
+    <ul>
+        <li class="dropdown">
+            <div id="dropdown-btn">
+                <img src="/public/icons/cloud.svg" alt="dropdown-button">
+                <i>{{ session.authInfo.friendly }}</i>
+            </div>
+            <div class="hamburger-icon">
+                <span></span>
+                <span></span>
+                <span></span>
+            </div>
+            <ul class="dropdown-menu" id="dd-menu">
+                <a href="/logout">
+                    <li><img src="/public/icons/logout.svg">Logout</li>
+                </a>
+                <li><i>Logged in as {{ session.username }}</i></li>
+            </ul>
+        </li>
+    </ul>
+</nav>
+
+<script>
+    document.addEventListener('click', function () {
```

Comment:
Not sure why this listener is added to the document and not the element being targeted

Reply:
aah i see, to hide the drop down menu when we click anywhere on the page, correct?

Reply:
Now I'm confused, since the drop down disappears when it's not hovered on, why is this needed?